app/components/ExtraSettings.vue.ts

```diff
@@ -61,7 +59,11 @@ export default class ExtraSettings extends Vue {
     this.cacheUploaderService.uploadCache().then(file => {
       electron.remote.clipboard.writeText(file);
       alert(
-        $t('Your cache directory has been successfully uploaded.  The file name %{file} has been copied to your clipboard.', { file })
+        $t(
+          // tslint:disable-next-line:max-line-length
```

Comment:
also this, we typically just use string concatenation in these cases which i think is a lot cleaner/more sustainable than selectively disabling rules

Reply:
+1 lets just concat.  I wish there was a tool that did this automatically.  Maybe there is.